draft-pauly-ohai-svcb-config.md

```diff
@@ -168,34 +180,42 @@ still need to ensure that they are not being targeted with unique
 key configurations that would reveal their identity. See {{security}} for
 more discussion.
 
-### Handling Oblivious DoH Configurations
+# Configuration Well-Known URI
+
+Once a client has discovered that a service is available as an oblivious
+target, it needs to know the key configuration before sending oblivious
+requests.
```

Comment:
```suggestion
Clients that know a service is available as an oblivious target, e.g., 
either via discovery through the "oblivious" parameter in a SVCB or HTTPS
record, or by configuration, need to know the key configuration before sending
oblivious requests.
```

Since the URI is separate from the discovery mechanism and could be used even for manually configured client/target arrangements.